src/presentation/wallet/send/end-of-flow.tsx

```diff
@@ -51,15 +51,15 @@ const EndOfFlow: React.FC<EndOfFlowProps> = ({
       const txid = await broadcastTx(explorerURL, tx);
       history.push({
         pathname: SEND_PAYMENT_SUCCESS_ROUTE,
-        state: { txid },
+        state: { txid }
       });
-    } catch (error) {
+    } catch (error: any) {
       return history.push({
         pathname: SEND_PAYMENT_ERROR_ROUTE,
         state: {
           tx: tx,
-          error: error.message,
-        },
+          error: error?.message || error
```

Comment:
If we expect a string (error.message), what are probabilities error is a string already, most likely is an object or undefined? Would be better to JSON.stringify?